cypress/integration/scenarios.spec.ts

```diff
@@ -0,0 +1,83 @@
+/// <reference types="cypress" />
+
+import scenarios from '../../src/assets/data/scenarios/scenarios.json'
+
+const scenariosKeys = Object.keys(scenarios);
+// testing all the scenarios would be extremely long
+const firstTenScenarios = scenariosKeys.slice(0, 10);
```

Comment:
There're **237** scenarios, running test on all of them will be time consuming, but if it' necessary we can remove this line and directly use `scenariosKeys` on line 49.